### ISSUE TITLE
tokenizer output is different from teacher's tokenizer

### ISSUE THREAD

**Issue by tagucci** [BandaiNamcoResearchInc/DistilBERT-base-jp#2]
Hi, 
Following [huggingface's model page](https://huggingface.co/bandainamco-mirai/distilbert-base-japanese), I download DistilBERT-base-jp model. My question is tokenizer returns different output compared with teacher's tokenizer ([cl-tohoku/bert-base-japanese-whole-word-masking](https://huggingface.co/cl-tohoku/bert-base-japanese-whole-word-masking)).

```python
from transformers import AutoTokenizer

teacher_tokenizer = AutoTokenizer.from_pretrained("cl-tohoku/bert-base-japanese-whole-word-masking")
tokenizer = AutoTokenizer.from_pretrained("bandainamco-mirai/distilbert-base-japanese")

print(teacher_tokenizer.tokenize("これはトークナイザーのテストです。"))
# ['これ', 'は', 'トーク', '##ナイ', '##ザー', 'の', 'テスト', 'です', '。']
print(tokenizer.tokenize("これはトークナイザーのテストです。"))
# ['これ', '##は', '##トー', '##ク', '##ナイ', '##ザー', '##の', '##テスト', '##で', '##す', '。']
```
My environment is
- ubuntu 16.04
- python 3.7.7
- pytorch 1.5.0
- transformers 2.11.0

Is this behavior correct?

**Comment by saurabhkumar8112** [comment]
Yeah, this doesn't seem right. On another note `cl-tohoku/bert-base-japanese-whole-word-masking` and `cl-tohoku/bert-base-japanese are identical`.
Would be nice if the authors clarify if the tokenization technique is different than standard BERT.
